<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="6" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-05-21T22:39:16 IST" name="Suite" finished-at="2025-05-21T22:39:46 IST" duration-ms="30764">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidationsTest.LoginErrorValidation()[pri:0, instance:SeleniumFramework.Design.ErrorValidationsTest@4c012563]" name="LoginErrorValidation" class="SeleniumFramework.Design.ErrorValidationsTest"/>
      </group> <!-- ErrorHandling -->
      <group name="Purchase">
        <method signature="SubmitOrderTest.submitOrder(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:SeleniumFramework.Design.SubmitOrderTest@34129c78]" name="submitOrder" class="SeleniumFramework.Design.SubmitOrderTest"/>
      </group> <!-- Purchase -->
    </groups>
    <test started-at="2025-05-21T22:39:16 IST" name="Submit Order Test" finished-at="2025-05-21T22:39:35 IST" duration-ms="18925">
      <class name="SeleniumFramework.Design.SubmitOrderTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:SeleniumFramework.Design.SubmitOrderTest@34129c78]" started-at="2025-05-21T22:39:16 IST" name="launchApplication" finished-at="2025-05-21T22:39:20 IST" duration-ms="4293" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:SeleniumFramework.Design.SubmitOrderTest@34129c78]" started-at="2025-05-21T22:39:20 IST" name="submitOrder" data-provider="getData" finished-at="2025-05-21T22:39:23 IST" duration-ms="3319" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[xyz6715@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Xyz@_12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ZARA COAT 3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:SeleniumFramework.Design.SubmitOrderTest@34129c78]" started-at="2025-05-21T22:39:23 IST" name="teardown" finished-at="2025-05-21T22:39:23 IST" duration-ms="141" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:SeleniumFramework.Design.SubmitOrderTest@34129c78]" started-at="2025-05-21T22:39:23 IST" name="launchApplication" finished-at="2025-05-21T22:39:26 IST" duration-ms="2127" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:SeleniumFramework.Design.SubmitOrderTest@34129c78]" started-at="2025-05-21T22:39:26 IST" name="submitOrder" data-provider="getData" finished-at="2025-05-21T22:39:29 IST" duration-ms="3406" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[abcrio@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Abc@_12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ADIDAS ORIGINAL]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:SeleniumFramework.Design.SubmitOrderTest@34129c78]" started-at="2025-05-21T22:39:29 IST" name="teardown" finished-at="2025-05-21T22:39:29 IST" duration-ms="150" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:SeleniumFramework.Design.SubmitOrderTest@34129c78]" started-at="2025-05-21T22:39:29 IST" name="launchApplication" finished-at="2025-05-21T22:39:34 IST" duration-ms="4557" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="OrderHistoryPage()[pri:0, instance:SeleniumFramework.Design.SubmitOrderTest@34129c78]" started-at="2025-05-21T22:39:34 IST" name="OrderHistoryPage" finished-at="2025-05-21T22:39:34 IST" depends-on-methods="SeleniumFramework.Design.SubmitOrderTest.submitOrder" duration-ms="764" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryPage -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:SeleniumFramework.Design.SubmitOrderTest@34129c78]" started-at="2025-05-21T22:39:34 IST" name="teardown" finished-at="2025-05-21T22:39:35 IST" duration-ms="92" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- SeleniumFramework.Design.SubmitOrderTest -->
    </test> <!-- Submit Order Test -->
    <test started-at="2025-05-21T22:39:35 IST" name="Error Validations Test" finished-at="2025-05-21T22:39:46 IST" duration-ms="11487">
      <class name="SeleniumFramework.Design.ErrorValidationsTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:SeleniumFramework.Design.ErrorValidationsTest@4c012563]" started-at="2025-05-21T22:39:35 IST" name="launchApplication" finished-at="2025-05-21T22:39:37 IST" duration-ms="1893" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="LoginErrorValidation()[pri:0, instance:SeleniumFramework.Design.ErrorValidationsTest@4c012563]" started-at="2025-05-21T22:39:37 IST" name="LoginErrorValidation" finished-at="2025-05-21T22:39:37 IST" duration-ms="516" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email  password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email  password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at SeleniumFramework.Design.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:24)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:SeleniumFramework.Design.ErrorValidationsTest@4c012563]" started-at="2025-05-21T22:39:37 IST" name="teardown" finished-at="2025-05-21T22:39:37 IST" duration-ms="126" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:SeleniumFramework.Design.ErrorValidationsTest@4c012563]" started-at="2025-05-21T22:39:37 IST" name="launchApplication" finished-at="2025-05-21T22:39:41 IST" duration-ms="3535" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="LoginErrorValidation()[pri:0, instance:SeleniumFramework.Design.ErrorValidationsTest@4c012563]" started-at="2025-05-21T22:39:41 IST" name="LoginErrorValidation" finished-at="2025-05-21T22:39:41 IST" duration-ms="433" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email  password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email  password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at SeleniumFramework.Design.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:24)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:SeleniumFramework.Design.ErrorValidationsTest@4c012563]" started-at="2025-05-21T22:39:42 IST" name="teardown" finished-at="2025-05-21T22:39:42 IST" duration-ms="135" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:SeleniumFramework.Design.ErrorValidationsTest@4c012563]" started-at="2025-05-21T22:39:42 IST" name="launchApplication" finished-at="2025-05-21T22:39:44 IST" duration-ms="2065" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:SeleniumFramework.Design.ErrorValidationsTest@4c012563]" started-at="2025-05-21T22:39:44 IST" name="ProductErrorValidation" finished-at="2025-05-21T22:39:46 IST" duration-ms="2237" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:SeleniumFramework.Design.ErrorValidationsTest@4c012563]" started-at="2025-05-21T22:39:46 IST" name="teardown" finished-at="2025-05-21T22:39:46 IST" duration-ms="133" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- SeleniumFramework.Design.ErrorValidationsTest -->
    </test> <!-- Error Validations Test -->
  </suite> <!-- Suite -->
</testng-results>
